recipes/openssh/files/init

```diff
@@ -1,6 +1,8 @@
 #! /bin/sh
 set -e
 
+HOST_KEY_TYPES="rsa dsa"
```

Comment:
why two different defaults? Also, I think we should just drop the insecure dsa key completely. no need to provide bad defaults. Rsa should also only be kept with minimum 4096bit size